### PR TITLE
Issue #3. Setup for Apache + PHP 7

### PR DESCRIPTION
**Pull request by johnrosswvsu**
Updated Read Me File for Apache + PHP 7 setup.

Updated docker-composer.yml file to use Apache.

Removed Nginx setup and add Apache set-up.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -2,13 +2,13 @@
 
 Express and simple docker setup for all your PHP development.
 
-### Nginx + PHP
+### Apache + PHP
 
-This branch is still on progress. For more stable, use the nginx branches.
+This branch is still on progress. For more stable, use the apache branches.
```
Comment (Luukyb) [inline README.md]: Please delete this line.

**Review by Luukyb — COMMENTED** [review]
The Dockerfile seems wrong. It still uses php:7-fpm.
It should be using "FROM php:7-apache".

You also need RUN a2enmod rewrite to enable the rewrite module. RUN a2ensite 000-default to enable the host.

When testing on your local can you try to remove the existing containers for the folder "docker-compose rm", then "docker-compose up -d --force-recreate", then just to make sure "docker-compose build --no-cache --".

When opening your browser on Expresso PHP, you should see the PHP info and mention of PHP 7 running on Apache.
Thanks!

File: README.md (hunk 1 above)
Comment (johnrosswvsu) [reply README.md]: Does this mean I have to remove this line also:


`
### PHP 7 or PHP 5?
You can switch between the latest release of PHP 7 or the latest of PHP 5, by
changing the first line of the file "docker/php/Dockerfile".
```
FROM php:5-fpm`

File: docker/php/Dockerfile [hunk 2]
```diff
@@ -16,7 +20,7 @@ RUN { \
         echo 'opcache.memory_consumption=128'; \
         echo 'opcache.interned_strings_buffer=8'; \
         echo 'opcache.max_accelerated_files=4000'; \
-        echo 'opcache.revalidate_freq=0'; \
+        echo 'opcache.revalidate_freq=60'; \
```
Comment (Luukyb) [inline docker/php/Dockerfile]: Can we keep zero? This parameter to 60 is great in production, but not in local. You would not see your changes immediately.

Comment (johnrosswvsu) [reply docker/php/Dockerfile]: Updated.

**Pull request merged**